spec/acceptance/zabbix_template_host_spec.rb

```diff
@@ -15,7 +15,7 @@ class { 'apache':
         include postgresql::server
 
         class { 'zabbix':
-          zabbix_version   => '4.4',
```

Comment:
since we still support zabbix 4.4 in the module: can you copy the whole test and use `5.0` is the second block? so we test both versions

Reply:
Do you think it's wise to test for the most versions in a do loop? Like i did in some specs. Probably from 4.0 and upwards?

Reply:
that would even be better. but I don't know if they all will pass on the first run. If you like you can give it a try, otherwise that might be a good thing for another PR.